deepseq.cabal

```diff
@@ -53,6 +53,11 @@ library
     if impl(ghc < 7.6)
       build-depends: ghc-prim == 0.2.*
 
+  if impl(ghc >= 7.8)
+    other-extensions:
+      ScopedTypeVariables
+      Trustworthy
```

Comment:
Ack, I misread again. I was looking at the `other-extensions` for the test suite, not the library, so my above comment doesn't really make sense.

So it's really only the placement of `ScopedTypeVariables` in this list that would need to change. Sorry for the confusion.